magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationTraverser.kt

```diff
@@ -16,6 +17,14 @@ class NavigationTraverser(private val root: Journey<*>) {
     return globalBackStack.filter { it.view != null }
   }
 
+  fun logGlobalBackStack() {
+    val navHistory = getGlobalBackStack().joinToString(" -> ") {
+      val type = if (it is Journey<*>) { "(Journey) " } else { "(Step) " }
```

Comment:
Minor: I would like to avoid having references to our concrete implementations (`Journey` and `Step`) as much as possible. We should consider checking for the existence of a navigator instead, but if that's too much work, then nevermind

Reply:
Done.